.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm test
+npm run build
+git add ./dist/main/index.js
```

Comment:
Husky hooks can be added to package.json

```
"husky": {
    "hooks": {
      "pre-commit": "npm test && npm run build && git add ./dist/main/index.js"
    }
  }
```

Reply:
Why do we need to use a precommit hook to add a file?

Reply:
@dpgraham this hook is to ensure that the updated "compiled" version of the action in `dist/main/index.js` (requirement for github actions to have no dependencies out of repository checkout).

Is the `.husky/pre-commit` approach not recommended ? I used that since it's as husky setup itself,